cfgov/jinja2/v1/_includes/organisms/full-width-text.html

```diff
@@ -87,8 +87,8 @@
         {% if 'content' in block.block_type %}
             <div class="m-full-width-text
                         {{ 'm-full-width-text__sans' if
-                        full_width_sans else '' }}">
-                {{ block.value.source | safe }}
+                        global_dict.full_width_sans else '' }}">
+                {{ render_stream_child(block) }}
             </div>
```

Comment:
There is a known issue in wagtail that rendering that extra `div` with the `rich-text` class is a pain for developers and they are looking to add a flag to turn this off https://github.com/torchbox/wagtail/issues/1214. In the meantime we'll expect to see this until the fix has been released.